src/core/vdom/helpers/normalize-scoped-slots.js

```diff
@@ -60,9 +61,10 @@ function normalizeScopedSlot(normalSlots, key, fn) {
     res = res && typeof res === 'object' && !Array.isArray(res)
       ? [res] // single vnode
       : normalizeChildren(res)
+    let vnode: VNode = res && res[0]
```

Comment:
shouldn't we keep the length check of 1?
```suggestion
    let vnode: VNode = res && res.length === 1 && res[0]
```

Reply:
This conversation was marked as resolved, but the suggested change didn't make it into the MR, and possibly caused #12102.